Doc/howto/unicode.rst

```diff
@@ -41,8 +41,9 @@ but these are two different characters that have different meanings.
 
 The Unicode standard describes how characters are represented by
 **code points**.  A code point value is an integer in the range 0 to
-0x10FFFF (about 1.1 million values, with some 110 thousand assigned so
-far).  In the standard and in this document, a code point is written
+0x10FFFF (about 1.1 million values, the actual number assigned is
+`less <http://www.unicode.org/versions/latest/#Summary>`_ than that).
```

Comment:
I think the link should be on the words "actual number assigned" instead. Also, it should be `https://`.

Reply:
Ok. 

The issue is, though, that https://www.unicode.org/versions/latest/#Summary currently redirects to http://www.unicode.org/versions/Unicode13.0.0/#Summary without `https://`. I still think that using the [latest link](https://www.unicode.org/versions/latest/#Summary) is better since it does not need a change to the HOWTO with every other update of unicode (provided the Unicode consortium maintains their documentation format).